robottelo/datafactory.py

```diff
@@ -37,18 +37,22 @@ def func_wrapper(*args, **kwargs):
         dataset = func(*args, **kwargs)
         if isinstance(dataset, dict):
             # New UI tests are written using pytest, update dict to support pytest's parametrize
-            if 'ui' in args or kwargs.get('interface') == 'ui' and settings.webdriver == 'chrome':
+            if (
+                'ui' in args
+                or kwargs.get('interface') == 'ui'
+                and settings.robottelo.webdriver == 'chrome'
```

Comment:
Are some people using firefox for their run?

Reply:
likely yes